anyway/views/safety_data/sd_utils.py

```diff
@@ -0,0 +1,212 @@
+from anyway.views.safety_data.involved_query import InvolvedQuery
+
+
+def sd_involved_query_from_anyway_tables():
+    S1: Streets = aliased(Streets)
+    S2: Streets = aliased(Streets)
+    res = (
+        db.session.query(Involved,
+                         AccidentMarkerView,
+                         City,
+                         InjurySeverity,
+                         Streets,
+                         Sex,
+                         RoadLight,
+                         RoadType,
+                         AgeGroup,
+                         )
+        .join(AccidentMarkerView,
+              and_(Involved.provider_code == AccidentMarkerView.provider_code,
+                   Involved.accident_id == AccidentMarkerView.id,
+                   Involved.accident_year == AccidentMarkerView.accident_year))
+        .join(City, AccidentMarkerView.yishuv_symbol == City.yishuv_symbol)
+        .join(InjurySeverity, and_(Involved.injury_severity == InjurySeverity.id,
+                                   Involved.accident_year == InjurySeverity.year,
+                                   Involved.provider_code == InjurySeverity.provider_code))
+        .join(S1, and_(AccidentMarkerView.street1 == S1.street,
+                       AccidentMarkerView.yishuv_symbol == S1.yishuv_symbol))
+        .join(S2, and_(AccidentMarkerView.street2 == S2.street,
+                       AccidentMarkerView.yishuv_symbol == S2.yishuv_symbol))
+        .join(Sex, and_(Involved.sex == Sex.id,
+                        Involved.accident_year == Sex.year,
+                        Involved.provider_code == Sex.provider_code))
+        .join(RoadLight, and_(AccidentMarkerView.road_light == RoadLight.id,
+                        AccidentMarkerView.accident_year == RoadLight.year,
+                        AccidentMarkerView.provider_code == RoadLight.provider_code))
+        .join(RoadType, and_(AccidentMarkerView.road_type == RoadType.id,
+                        AccidentMarkerView.accident_year == RoadType.year,
+                        AccidentMarkerView.provider_code == RoadType.provider_code))
+        .join(AgeGroup, and_(Involved.age_group == AgeGroup.id,
+                        Involved.accident_year == AgeGroup.year,
+                        Involved.provider_code == AgeGroup.provider_code))
+        .with_entities(
+            Involved.id,
+            AccidentMarkerView.accident_year,
+            AccidentMarkerView.accident_timestamp,
+            City.heb_name,
+            S1.street_hebrew,
+            S2.street_hebrew,
+            RoadLight.road_light_hebrew,
+            RoadType.road_type_hebrew,
+            AccidentMarkerView.latitude,
+            AccidentMarkerView.longitude,
+            InjurySeverity.injury_severity_hebrew,
+            Sex.sex_hebrew,
+            AgeGroup.age_group_hebrew,
+        )
+        .limit(50).all()
+    )
+    return Response(json.dumps(res, default=str), mimetype="application/json")
+
+def sd_load_data():
```

Comment:
Since `SDAccident` and `SDInvolved` are related in foreign key, and we can't have an involved in DB w/o a related accident, then the deletion/insertion order should be changed to the following:
1. delete involved
2. delete accidents
3. insert accidents
4. insert involved

Otherwise I believe we can encounter foreign key errors in DB

Reply:
In addition, I will add temporary table, fill them, and then copy to the actual, in order to avoid downtime.

Reply:
I'm not sure temporary table is necessary.
Can we possible perform it all in the same transaction (like we [discussed here on CBS hebrew views](https://github.com/data-for-change/anyway/pull/2247#discussion_r973591896))?

If we have no choice, we can add temporary tables.

Reply:
Safety-data tables are loaded in a different connection / session, so no need for temporary tables.